code/game/turfs/simulated/mineral.dm

```diff
@@ -11,12 +11,8 @@
 /turf/simulated/floor/mineral/plasma
 	name = "plasma floor"
 	icon_state = "plasma"
-	floor_tile = /obj/item/stack/tile/mineral/plasma
```

Comment:
I know I should ask why, so I'll just ask. Why?

Also wow I never realized how bad this system was before.